docs/docs/components/migrating/README.md

```diff
@@ -0,0 +1,190 @@
+# Migrating from Legacy Actions to Component Actions
+
+- [Overview](#overview)
+- [Key Changes](#key-changes)
+- [Getting Started](#getting-started)
+- [Migration Example](#migration-example)
+  * [Legacy Code Example](#legacy-code-example)
+  * [Converting into a Basic Component-Based Action](#converting-into-a-basic-component-based-action)
+  * [Advanced: Using Async Options](#advanced--using-async-options)
+  * [Publishing and Using Actions](#publishing-and-using-actions)
+
+## Overview
+This document is for developers who created legacy actions in [Pipedream's UI](https://pipedream.com/actions). The purpose is to help users migrate legacy actions to Pipedream's new [component model](/components). There is currently no deprecation timeline for legacy actions, but developers are encouraged to migrate to the new model. 
+
+## Key Changes
+
+**Capture user input via `props` instead of `params`** 
+
+The component model does not support `params`. You need to migrate `params` references to `props`. Unlike `params`, `props` must be explicitlly declared and defined prior to using them in code (in the old model, an input form was automatically generated when  `params` were used in code -- `params` were not explicitly declared).
```

Comment:
extra "l" in "explicitlly".

Consider an em dash (Shift + Option + `-`) instead of double `--`.

Reply:
Extra space between "when" and `params`

Reply:
Worth including links to component API docs on props?

Reply:
done